README.md

```diff
@@ -0,0 +1,103 @@
+LTI Learner Profile Dashboard
+=============================
+
+Prerequisites
+-------------
+
+* python-dev
+* virtualenv
+
+Setting up the development server
+---------------------------------
+
+1. Install python dependencies:
+
+    ```bash
+    virtualenv .virtualenv
+    . .virtualenv/bin/activate
+    pip install -r requirements/base.txt
+    ```
+
+1. Create `app/local_settings.py`, and set the sensitive settings:
+
+    ```python
+    SECRET_KEY = 'SET ME'
+    LTI_CLIENT_KEY = 'SET ME'
+    LTI_CLIENT_SECRET = 'SET ME'
+    PASSWORD_GENERATOR_NONCE = 'SET ME'
+
+    # Optional - can use default sqlite for dev
+    LPD_DB_PASSWORD = 'SET ME'
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.mysql',
+            'NAME': 'lti_lpd,
```

Comment:
Nit: `lti_lpd` in both `NAME` and `USER` needs closing apostrophe.

Reply:
@tomaszgy Nice catch, thanks :)